server/server.go

```diff
@@ -1378,3 +1376,15 @@ func (s *Server) SaveTTLConfig(data map[string]interface{}, ttl time.Duration) e
 	}
 	return nil
 }
+
+func stringsEqual(a, b []string) bool {
```

Comment:
Judge the situation where `len` is all 0, then we can still use `reflect.DeepEqual` directly?

Reply:
Then it will be `if len(a) == 0 && len(b) == 0 || reflect.DeepEqual(a, )`, looks not very elegant I think.